.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    name: Build shared libraries
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Install dependencies
+        shell: bash
```

Comment:
Fine as is, but if you wish to you can set the shell for the whole workflow with this, at the top:
```
defaults:
  run:
    shell: bash
```
See: https://docs.github.com/en/actions/using-jobs/setting-default-values-for-jobs#example-set-the-default-shell-and-working-directory

The docs don't mention it but you can also set shell flags if you wish, e.g.: 
```
defaults:
  run:
    shell: bash -euxo pipefail {0}
```

Reply:
Ah, nice to know. Thanks! I updated accordingly.